javascript-loops/main.js

```diff
@@ -0,0 +1,65 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  const numbers = [];
+  let currentNumber = 1;
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  } return numbers;
+}
+console.log('Numbers to ten ', getNumbersToTen());
+
+function getEvenNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  } return evenNumbers;
+}
+console.log('Get even numbers to twenty ', getEvenNumbersToTwenty());
+
+function repeatWord(word, times) {
+  let count = 1;
+  let repeated = ' ';
+  while (count < times) {
```

Comment:
This should be `<=`. If times is 10, the loop would only run 9 times, but we want it to run 10 times